components/molecule/thumbnail/src/tablet-mobile.scss

```diff
@@ -0,0 +1,26 @@
+@import '~@schibstedspain/sui-theme/lib/settings-compat-v7/index';
+
+$sizes-mobile-molecule-thumbnail: (
+  large: 96px,
+  medium: 72px,
+  small: 40px,
+  xsmall:  32px
+) !default;
+
+.sui-MoleculeThumbnail {
+  @include media-breakpoint-down(m) {
```

Comment:
this mixin is forbidden for new implementations, please only use `media-breakpoint-up` for now on as decided in this [meeting](https://confluence.schibsted.io/display/ST/Breakpoints+-+Frontend)